### PR TITLE
Debug tpdm models

### PR DESCRIPTION
**Pull request by zaharenka**
@holdorph I have almost successful dbt run for tpdm models after this patch. Only one model still require raw data to be processed: `models/staging/tpdm/base/base_tpdm__evaluation_elements.sql`

image

**UPD: evaluation_elements models removed** 

`dbt run --select tag:tpdm`

image

### CODE REVIEW

File: dbt_project.yml [hunk 1]
```diff
@@ -36,7 +36,6 @@ models:
         stage: 
           +materialized: table
       tpdm:
-        +enabled: "{{ var('edu:tpdm:enabled', false) }}"
```
Comment (zaharenka) [inline dbt_project.yml]: If we want, we can play with variable in order to have control of materializing whether to include them in dbt run or not. But I don't feel it really needed.

Comment (holdorph) [reply dbt_project.yml]: Please remove this change from the PR for now.  I'd like to get the other change from this PR merged, but I'm not sure Erik will agree to this dbt_project.yml change, but the ::personId change is needed for certain.

**Review by holdorph — CHANGES_REQUESTED** [review]

File: dbt_project.yml (hunk 1 above)
Comment (holdorph) [reply dbt_project.yml]: I did also ask Erik about this though, will update my comment here, if I learn anything that changes the request above to remove this change.

Comment (zaharenka) [reply dbt_project.yml]: Left the variable as this. Will try my idea to define +enable in models level, but not for folder level

Comment (zaharenka) [reply dbt_project.yml]: If I find a solution, I'll produce new PR.

Comment (holdorph) [reply dbt_project.yml]: the code that goes into main, therefore the code that goes into the ed analytics PR needs to have this in there, we can just comment out the line locally for now, but that should not be removed for the code that is in main.

**Review by holdorph — CHANGES_REQUESTED** [review]
Please remove the base_tpdm__evaluation_elements.sql and corresponding stage files.

File: dbt_project.yml (hunk 1 above)
Comment (zaharenka) [reply dbt_project.yml]: Ok, got it. Let's leave the code as it was before - done.

**Review by holdorph — APPROVED** [review]

**Review by groybal — APPROVED** [review]

**Pull request merged**